Makefile

```
@@ -0,0 +1,141 @@
+# Executables
+EXEC_NODEJS = $(DOCKER_COMPOSE) exec -T nodejs
+PNPM = $(EXEC_NODEJS) pnpm
+PNPM_RUN = $(PNPM) run
+HOST_PNPM = pnpm
+HOST_NPX = npx
+GIT = git
+
+# Misc
+.DEFAULT_GOAL = help
+.RECIPEPREFIX +=
+.PHONY: $(filter-out node_modules,$(MAKECMDGOALS))
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)